integration_tests/src/main/python/asserts.py

```diff
@@ -220,6 +220,22 @@ def bring_back(spark):
             raise RuntimeError('Local Sort is only supported on a collect')
     return (bring_back, collect_type)
 
+def _sort_locally(from_cpu=None, from_gpu=None):
```

Comment:
please add a high-level comment to explain the algorithm here 

it will read easier if we had another function for a single result that we would apply to from_cpu and from_gpu to eliminate logic duplication